src/index.html

```diff
@@ -8,6 +8,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+
+      <div class="stars__row stars--0">
```

Comment:
not correct naming. you should use modifiers only with block/element which they modify. so it should be like ...stars stars--0